submission/proxy_test.go

```diff
@@ -111,6 +112,22 @@ func TestProxyRefreshRootsErr(t *testing.T) {
 	}
 }
 
+func updateFile(name string, data string) error {
+	f, err := os.OpenFile(name, os.O_WRONLY, 0644)
```

Comment:
I don't think this does what you think it does. O_WRONLY will start writing from the start of the file, overwriting whatever is already there. If you write fewer bytes than the length of the file, whatever was already in the file will be left at the end.

Consider using `ioutil.WriteFile(name, []byte(data), 0644)` instead.

Reply:
Done, thanks